plugins/ui/nvim-highlight-colors.nix

```
@@ -0,0 +1,96 @@
+{
+  lib,
+  helpers,
+  config,
+  pkgs,
+  ...
+}:
+
+let
+  inherit (helpers.defaultNullOpts)
+    mkBool
+    mkEnum
+    mkListOf
+    mkStr
+    mkStr'
+    ;
+in
+helpers.neovim-plugin.mkNeovimPlugin config {
```

We (very) recently dropped the `config` arg:
```suggestion
lib.nixvim.neovim-plugin.mkNeovimPlugin {
```